columnflow/tasks/framework/base.py

```diff
@@ -710,6 +711,19 @@ def store_parts(self):
 
         return parts
 
+    def find_keep_columns(self: ConfigTask, collection: ColumnCollection) -> set[Route]:
+        """
+        Returns a set of :py:class:`Route` objects describing columns that should be kept given a
+        type of column *collection*.
+
+        :param collection: The collection to return.
+        :return: A set of :py:class:`Route` objects.
+        """
+        if collection == ColumnCollection.MANDATORY_COFFEA:
+            return set(Route(c) for c in mandatory_coffea_columns)
```

Comment:
I think you never initialized the `MANDATORY_COFFEA` class attribute

Reply:
True. Added :+1: